src/osha/oira/client/browser/client.py

```diff
@@ -61,6 +58,30 @@ def results(self):
 
         if ticket != expected:
             raise Unauthorized("Invalid ticket")
+        return user_id
+
+    def __call__(self):
+        """Returns a json meant to be consumed by pat-autosuggest.
+
+        The json lists container that will be used to generate "mailing
+        lists"
+        """
```

Comment:
What about adding the validate ticket in the `__call__`?

Reply:
I assumed @thet put it into `results` for a reason, but maybe I should have checked...

Reply:
Moved it.